synchronizer/views/auth.py

```diff
@@ -63,6 +62,11 @@ def oauth_callback(provider):
     """
     oauth = OAuthSignIn.get_provider(provider)
     user_info = oauth.callback()
+
+    if current_app.config['ACCESS_EMAIL'] not in user_info['email']:
+        flash('Only employees with email of Ventor Tech can get access!', 'error')

```

Comment:
Update message to be more abstract (**remember**: this app is open source, so anyone can install and use it).